### PR TITLE
docs/design: update the proposal of `column type change`

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

Describe the features that are not currently supported.

### Check List 

Tests 

- No code

### Release note 

- `No release note`.

### CODE REVIEW

File: docs/design/2020-07-07-change-column-types.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ This requires changing the state of `colA` and `idxA` to `StatePublic`, and then
 ### Compatibility issues with MySQL
 
 * Considering the complexity of supporting clustered-index and currently TiDB does not fully support clustered-index, so temporarily this feature does not support type modification for columns with the primary key.
+* In addition, considering the complexity and relevance of some feature support, we temporarily do not support the three features of partition table, generated column, and expression index in the first implementation.
```
Comment (AilinKid) [inline docs/design/2020-07-07-change-column-types.md:72]: In addition, given the complexity and relevance of some features, we temporarily do not support the column-type-change on partition table, generated column, and expression index in the first stage of implementation.

Comment (XuHuaiyu) [inline docs/design/2020-07-07-change-column-types.md:72]:
```suggestion
* In addition, we temporarily do not support the three features of partition table, generated column, and expression index in the first implementation.
```

File: docs/design/2020-07-07-change-column-types.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ This requires changing the state of `colA` and `idxA` to `StatePublic`, and then
 ### Compatibility issues with MySQL
 
 * Considering the complexity of supporting clustered-index and currently TiDB does not fully support clustered-index, so temporarily this feature does not support type modification for columns with the primary key.
+* In addition, we temporarily do not support the three features of partition table, generated column, and expression index in the first implementation.
```
Comment (AilinKid) [inline docs/design/2020-07-07-change-column-types.md:72]:
```suggestion
* In addition, we temporarily do not support column type change on three features of partition table, generated column, and expression index in the first stage of implementation.
```

**Pull request merged**